Corona-Warn-App/src/main/res/layout/contact_diary_onboarding_fragment.xml

```diff
@@ -0,0 +1,221 @@
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_normal"
+                    android:focusable="true"
+                    app:body="@{@string/contact_diary_onboarding_functionality_fifth_section}"
+                    app:icon="@{@drawable/ic_contact_diary_file}"
+                    app:layout_constraintEnd_toStartOf="@+id/guideline_end"
+                    app:layout_constraintStart_toStartOf="@+id/guideline_start"
+                    app:layout_constraintTop_toBottomOf="@+id/contact_diary_onboarding_fourth_section" />
+
+
+                <include
+                    android:id="@+id/contact_diary_onboarding_privacy_card"
+                    layout="@layout/include_contact_diary_privacy_card"
+                    android:layout_width="@dimen/match_constraint"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_normal"
+                    android:focusable="true"
+                    app:layout_constraintEnd_toStartOf="@+id/guideline_end"
+                    app:layout_constraintStart_toStartOf="@+id/guideline_start"
+                    app:layout_constraintTop_toBottomOf="@+id/contact_diary_onboarding_fifth_section" />
+
+                <View
+                    android:id="@+id/contact_diary_onboarding_first_divider"
+                    android:layout_width="@dimen/match_constraint"
+                    android:layout_height="1dp"
+                    android:layout_marginTop="@dimen/spacing_small"
+                    android:background="?android:attr/listDivider"
+                    app:layout_constraintEnd_toStartOf="@+id/guideline_end"
+                    app:layout_constraintStart_toStartOf="@+id/guideline_start"
+                    app:layout_constraintTop_toBottomOf="@+id/contact_diary_onboarding_privacy_card" />
+
+
+                <TextView
+                    android:id="@+id/contact_diary_onboarding_privacy_information"
+                    style="@style/subtitle"
+                    android:layout_width="@dimen/match_constraint"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_small"
+                    android:focusable="true"
+                    android:text="@string/contact_diary_onboarding_legal_information"
+                    app:layout_constraintEnd_toEndOf="@id/guideline_end"
+                    app:layout_constraintStart_toStartOf="@id/guideline_start"
+                    app:layout_constraintTop_toBottomOf="@+id/contact_diary_onboarding_first_divider"
+                    tools:text="@string/contact_diary_onboarding_legal_information" />
+
+                <View
+                    android:id="@+id/contact_diary_onboarding_second_divider"
+                    android:layout_width="@dimen/match_constraint"
+                    android:layout_height="1dp"
```

Comment:
do we have any dimens setup for spacers like this? otherwise you could create one